src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -158,7 +158,7 @@ class SignInModal extends React.Component {
             <div className="help-info">
               <h4>Having trouble signing in?</h4>
               <p>
-                <a href="/faq/" target="_blank">
+                <a href="/sign-in-faq" target="_blank">
```

Comment:
Because this affects Vets.gov and it’s FAQ isn’t moving, I think we can leave the src directory alone for now and just log an `aliases` property on the `va-gov` FAQ. 

This done raise the question of how we’ll in the end update the links in the src directory, but I think we can do that all at once with that mapping script.